app/frontend/src/service/conversations.ts

```diff
@@ -0,0 +1,48 @@
+import {
+  CreateGroupChatReq,
+  GetGroupChatMessagesReq,
+  GroupChat,
+  ListGroupChatsReq,
+  Message,
+  SendMessageReq,
+} from "../pb/conversations_pb";
+import client from "./client";
+import { User } from "../pb/api_pb";
+
+export async function listGroupChats(): Promise<GroupChat.AsObject[]> {
```

Comment:
Do you think we should include brief docstrings as with the other service functions, or do you think it's pointless?

Reply:
Maybe it's pointless since the documentation should really be in the `.proto` files

Reply:
Most of the time these methods link on directly to the generated methods. I seldom write comments on functions and then only on functions that do business logic. Like calculations or string arithmetic. For service interfaces and views I do not think it is necessary. Except maybe for libraries or functions that will be used often like those in the components folder.